### ISSUE TITLE
Cypress UI stops to display testcases and displays an internal error

### ISSUE THREAD

**Issue by karevn** [cypress-io/cypress#4072]
### Current behavior:

When using the UI, Cypress sometimes displays this error and stops displaying testcases:
![cypress](https://user-images.githubusercontent.com/7992040/56946368-56844280-6b43-11e9-9b08-d349e71eadc3.png)

### Steps to reproduce: (app code and test code)

* Have Cypress + cypress cucumber preprocessor
* Open some test suite and run it several times by manually refreshing + updating tests (it's sporadic)

### Versions

Cypress 3.2, latest Cucumber preprocessor, MacOS Mojave 10.4.4

**Comment by GordonTester** [comment]
I see this every now and then if I run a series of tests in succession - from what I remember, the GUI doesn't cope well with large runs and as really only meant for debugging a test at a time.

**Comment by karevn** [comment]
I suppose it's an error still.

BTW, I was able to reproduce it by clicking "rerun tests"  circle button quickly several times.

**Comment by jennifer-shehane** [comment]
This is strange because the stacktrace seems to indicate this is coming from `_reRun` where we explicitly return if Cypress is not defined. 

https://github.com/cypress-io/cypress/blob/develop/packages/runner/src/lib/event-manager.js#L322

**Comment by CoryDanielson** [comment]
The error is coming from cypress_runner.js line 60170

```js
/* 60169 */  $Cypress.prototype.stop = function() {
/* 60170 */    this.runner.stop(); // this line
/* 60171 */    this.cy.stop();
/* 60172 */    return this.action("cypress:stop");
/* 60173 */  };
```

Something is setting the runner to null. I did find `this.runner = null` within the same file, but I'm not sure if it could be something else

```js
$Cypress = (function() {
  function $Cypress(config) {
    if (config == null) {
      config = {};
    }
    this.cy = null;
    this.chai = null;
    this.mocha = null;
    this.runner = null;
```

**Comment by emilyrohrbough** [comment]
A safe-guard was added in https://github.com/cypress-io/cypress/pull/7984, which was released in [4.11.0](https://on.cypress.io/changelog#4-11-0) on Jul 21, 2020. Closing as fixed.
